srcpkgs/kicad/template

```diff
@@ -1,26 +1,26 @@
 # Template file for 'kicad'
 pkgname=kicad
-version=5.1.11
-revision=4
+version=6.0.1
+revision=1
 build_style=cmake
 build_helper=cmake-wxWidgets-gtk3
-configure_args="-DKICAD_BUILD_VERSION=${version} -DKICAD_SCRIPTING=ON
- -DKICAD_SCRIPTING_MODULES=ON -DKICAD_SCRIPTING_WXPYTHON=ON
- -DKICAD_SCRIPTING_ACTION_MENU=ON  -DBUILD_GITHUB_PLUGIN=ON -DKICAD_USE_OCE=OFF
+configure_args="-DKICAD_SCRIPTING=ON -DKICAD_SCRIPTING_MODULES=ON
+ -DKICAD_SCRIPTING_WXPYTHON=ON -DKICAD_SCRIPTING_ACTION_MENU=ON
  -DKICAD_SCRIPTING_PYTHON3=ON -DKICAD_SCRIPTING_WXPYTHON_PHOENIX=ON
  -DwxWidgets_CONFIG_EXECUTABLE=$WX_CONFIG
  -DKICAD_USE_OCC=$(vopt_if occt ON OFF) -DKICAD_SPICE=$(vopt_if spice ON OFF)"
-hostmakedepends="pkg-config swig wxWidgets-gtk3-devel python3"
+hostmakedepends="pkg-config swig wxWidgets-gtk3-devel python3 tar"
```

Comment:
What does it need tar for?

Just want to be sure it isn't downloading something during the build

Reply:
Iirc it errored when trying to unpack the source right after I had done an `xbps-src zap && xbps-src binary-bootstrap`, though I might be wrong, it's been more than a week.

What would it be downloading, though?

Reply:
That checks out, I had forgotten about the new requirement.